### PR TITLE
Doc about TLS security profile

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Doc about TLS security profile

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [ ] Bump-up library or tool used for development (does not change the final image)
- [ ] CI configuration change
- [ ] Konflux configuration change

### CODE REVIEW

**Comment by tisnik** [comment]
/override "Red Hat Konflux / ols-enterprise-contract / lightspeed-service"

File: README.md [hunk 1]
```diff
@@ -437,7 +437,28 @@ Depends on configuration, but usually it is not needed to generate or use API ke
 ## 8. Registering a new LLM provider
    Please look [here](https://github.com/openshift/lightspeed-service/blob/main/CONTRIBUTING.md#adding-a-new-providermodel) for more info.
 
-## 9. Fine tuning
+## 9. TLS security profiles
+   TLS security profile can be set for the service itself and also for any configured provider. To specify TLS security profile for the service, the following section can be added into `ols` section in the `olsconfig.yaml` configuration file:
+
+```
+  tlsSecurityProfile:
+    type: OldType
```
Comment (raptorsun) [inline README.md:445]: When setting the type to one of the 3 types OldType, IntermediateType, ModernType, is it still required to set the ciphers and minTLSVersion?

Comment (raptorsun) [reply README.md:445]: I think it better to have defaults on known types and use Custom type to set ciphers and minTLSVersion

Comment (tisnik) [reply README.md:445]: no, it is optional

Comment (tisnik) [reply README.md:445]: yes @raptorsun

**Comment by tisnik** [comment]
/override "Red Hat Konflux / lightspeed-service-on-pull-request"

**Pull request merged**